aardvark/manage.py

```diff
@@ -300,51 +232,54 @@ def _prep_accounts(account_names):
     Returns a list of account numbers
```

Comment:
The "while not" should be based on a signal or message instead, or on all threads exiting. It might make more sense to put the condition to die "UPDATE_DONE" inside of the updater threads as a class variable

Reply:
Deferring to #116